src/Bundle/DependencyInjection/WebpackExtension.php

```diff
@@ -72,7 +72,7 @@ private function getPlatformKey()
         if (strtoupper(substr(php_uname('s'), 0, 5)) === 'LINUX') {

```

Comment:
PHP_OS returns information about the machine the PHP binaries were compiled on; not the current system.